src/rebar3_autotest_prv.erl

```diff
@@ -70,20 +80,31 @@ remove_from_plugin_paths(State) ->
   end, PluginPaths),
   rebar_state:code_paths(State, all_plugin_deps, PluginsMinusAutotest).
 
+-spec
+should_check(Event) -> boolean() when
+    Event :: {AbsPathFile, Attributes},
+    AbsPathFile :: string(),
+    Attributes :: [atom()].
+should_check(_Event = {AbsPathFile, _Attributes}) ->
+  IncludeREs = lists:map(fun(S) -> {ok, MP} = re:compile(S), MP end, ?INCLUDE_FILE_PATTERNS),
```

Comment:
Does the compiler see `IncludeREs` as an invariant and cache this? Inquiring minds wanna know.